.github/workflows/release.yml

```
@@ -35,6 +35,24 @@ jobs:
           checksum_file=$(echo "$ARTIFACTS" | jq -r '.[] | select (.type=="Checksum") | .path')
           echo "::set-output name=hashes::$(cat $checksum_file | base64 -w0)"
 
+  publish:
+    # outputs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: git fetch --prune --unshallow
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+          check-latest: true
+      - uses: imjasonh/setup-ko@v0.6
```

```suggestion
      - uses: imjasonh/setup-ko@v0.6 # This installs the current latest release.
```

(Just to save me some thinking when I read this later 😄)